swagger.yaml

```diff
@@ -3786,6 +3786,64 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/ui-settings:
+    get:
+      tags:
+        - installer
+      description: Fetch cluster specific installer UI settings.
+      operationId: v2GetClusterInstallerUISettings
```

Comment:
I think you can remove `Installer` from this as well.

Reply:
I disagree with this, removing this tag sees the code generation placing the request objects under the operations package, somehow when it does this, you lose the ability to return a payload. 

So I will bve sticking with this tag for now.

Reply:
Do you have any suggestion on how this can be made to work or are we OK to keep the installer tag?

Reply:
```suggestion
      operationId: V2GetClusterUISettings
```